### PR TITLE
Buffing plasmafires [READY FOR REVIEW]

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
The aim of this PR is to bring plasmafires back to their former glory a tad bit. Currently, this only makes it so floors practically won't melt anymore, resulting in plasmafires lasting longer, and leaving hotter rooms behind for atmos to fix, instead of it fixing itself by spacing the hot air.

What is currently working:
- Floors are much, much harder to melt
- Airlocks and firedoors will now also melt from heat, be it fire or other hot gasses.
- Walls will now melt from hot enough fires.

Current temps required to melt:
- Walls: 9000 Kelvin
- Firelock: 15.000 Kelvin
- Reinforced Firelock: 20.000 Kelvin
- Airlocks: 1500 Kelvin
- Reinforced walls: 40.000 Kelvin
- Reinforced windows: 1300 Kelvin

**This will obviously require a testmerge**

## Why It's Good For The Game
Currently, almost all hot plasmafires die out prematurely because the floors will melt and are stopped by firelocks and walls. For example, this makes the main threat of a plasmafire the resulting holes into space, which usually won't amount to much.

## Images of changes
Only plating heat capacity changed:
image

With wallmelting and testing only values:
image

These two images will be replaced when I'll do full-scale testing with the actual values.
## Testing
See image

## Changelog
:cl:
tweak: Increased heat capacity for normal plating
tweak: Doors and firelocks can now melt due to high temperature gas
tweak: Walls can now melt from plasmafires with a high enough temperature.
/:cl:

### CODE REVIEW

**Comment by Dafrek** [comment]
This sounds great, making firelocks not completely immune to heat will mean that we no longer have to intentionally design toxins storage with a weak point so that plasma fires can escape into the rest of science. Having firelocks have a lower damage temperature than walls and floors would allow for a more natural spread of plasma fire while having less of an impact on performance at lower temperatures, although I can understand if meltable walls are not re added.

**Comment by DGamerL** [comment]
Aaaaa finally found the godforsaken button to make it a draft

**Comment by DGamerL** [comment]
While I fix the runtime I'd like to request a testmerge on this PR

File: code/modules/atmospherics/environmental/LINDA_fire.dm [hunk 1]
```diff
@@ -41,6 +41,12 @@
 		active_hotspot.just_spawned = (current_cycle < SSair.times_fired)
 			//remove just_spawned protection if no longer processing this cell
 		SSair.add_to_active(src, 0)
+	var/list/adjacent_dense_turfs = AdjacentTurfs(FALSE, TRUE, TRUE) //Checks adjacent  turfs for non-dense turfs, only in the 4 cardinal directions
```
Comment (S34NW) [inline code/modules/atmospherics/environmental/LINDA_fire.dm:44]: For non-dense turfs? Why is it called `adjacent_dense_turfs` if so? (I think the comment is wrong)

Comment (S34NW) [reply code/modules/atmospherics/environmental/LINDA_fire.dm:44]: Also there's an extra space in `adjacent  turfs`

**Review by S34NW — APPROVED** [review]

**Pull request merged**